packages/protocol/README.md

```diff
@@ -26,53 +26,19 @@ If a new contract needs to be deployed, create a migration file in the `migratio
 To apply any new migrations to a network, run:
 
 ```bash
-yarn run migrate -n NETWORK
+yarn run devchain:migrate -n NETWORK
 ```
 
 ### Accounts
 
-To give an account some gold, wrapped gold, and stable token, run:
-
-```bash
-yarn run faucet -n NETWORK -a ACCOUNT_ADDRESS
-```
-
-You can check balances by running:
-
-```bash
-yarn run get-balances -n NETWORK -a ACCOUNT_ADDRESS
-```
-
-You can run 'onlyOwner' methods via the [MultiSig](contracts/common/MultiSig.sol) by running:
-
-```bash
-yarn run govern -n NETWORK -c "stableToken.setMinter(0x1234)"
-```
-
-### Build artifacts
-
-When interacting with one of our Kubernetes-deployed networks, you can download the build artifacts to a local directory using:
-
-```bash
-yarn run download-artifacts -n NAME
-```
-
-You must run this before interacting with one of these networks to have the build artifacts available locally.
-
-If you changed the build artifacts (e.g. by running the `init-network`, `migrate`, or `upgrade` script), upload the new build artifacts with:
-
-```bash
-yarn run upload-artifacts -n NAME
-```
-
-By default, `NAME` will be set as `RELEASE_NAME`, `NAMESPACE_NAME`, `TESTNET_NAME` which you should have used with the same name in prior instructions. If you used separate names for the above, you can customize the run with the `-r -n -t` flags respectively.
+TBD
```

Comment:
What do you mean by TBD? Or should this whole section just be removed, given we have other tools that have replaced most (all?) of these legacy scripts?

Reply:
Left TBD as To be Discussed / Decided / Done later, but I will remove the whole section 😉